### PR TITLE
completed javascript-operators-and-expressions - Aaron He

### PR DESCRIPTION
**Pull request by AaronHe09**
image

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const width = 5;
+const height = 10;
+const area = width * height;
+console.log('Value of area: ', area);
+console.log('typeof area: ', typeof area);
+
+const bill = 11;
+const payment = 4;
+const change = bill - payment;
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:9]: Your `payment` should be larger than your `bill`. You'll also need to swap line 9. It should be `payment - bill`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just one small change requested here Aaron!

File: javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (AaronHe09) [reply javascript-operators-and-expressions/main.js]: image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**